pep/app/styles/_common.scss

```diff
@@ -1,6 +1,8 @@
 // ---------------------------
 // Vendor / Template
 // ---------------------------
+@import 'variables';
+@import 'variables-theme';
```

Comment:
theses cant be imported here, as its used across all themes, and both are already imported in `styles/app.scss` and `styles/themes/tomato.scss`.